index.html

```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>MyBrand</title>
+    <title>AnastaseTuyizere</title>
```

Comment:
Remove one  title here maybe you can remove ` <title>AnastaseTuyizere</title>`!

Reply:
yes, i will remove first title